### ISSUE TITLE
Disable animations with `prefers-reduced-motion`

### ISSUE THREAD

**Issue by flamber** [metabase/metabase#17726]
**Is your feature request related to a problem? Please describe.**
For accessibility reasons, animations should be disabled (or very limited) to honor browser setting.
Would also make Cypress CI testing faster and sometimes avoid race-conditions, since it's waiting for animations to complete.
And is also helpful on older/slower computers, where CPU spikes when loading a dashboard with a handful of loading-spinners.

**Describe the solution you'd like**
Honor the browser setting for `prefers-reduced-motion`
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

**Additional context**
General accessibility #1462
